runtime/doc/manpages/nvim.1.md

```diff
@@ -0,0 +1,261 @@
+<!---
+Should be generated using md2man
+https://github.com/sunaku/md2man
+-->
+
+# NVIM 1 "April 2015" "Neovim" "User Commands"
+
+## NAME
+
+nvim - Vim out of the box
+
+## SYNOPSIS
+
+`nvim` [*options*] [*file*]...  
+`nvim` [*options*] `-`  
+`nvim` [*options*] `-t` <*tag*>  
+`nvim` [*options*] `-t` [*errorfile*]
+
+## DESCRIPTION
+
+`nvim` is an extensible text editor based off of Vim.
```

Comment:
I would change "based off of Vim" to "based on Vim." The former sounds overly colloquial to me.